packages/salesforcedx-vscode-lwc/snippets/lwc.json

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
@muenzpraeger - What did we settle on for prettier? I'm asking because some of the snippets below look like they are using 4 spaces for the indentation (prettier's default is 2). I know that prettier would just do what it needs to do and format it but it would be nice to have the snippets be as close as possible.